CMakeLists.txt

```diff
@@ -137,7 +147,9 @@ install (FILES src/pipeline.h DESTINATION include/nanomsg)
 install (FILES src/survey.h DESTINATION include/nanomsg)
 install (FILES src/bus.h DESTINATION include/nanomsg)
 
-install (TARGETS nanocat RUNTIME DESTINATION bin)
+if (NN_TOOLS)
```

Comment:
It is, just not with this name. The question is whether it makes sense to keep the current flag strategy of a one-flag-per-tool basis, or to lump them all under a single `TOOLS` flag.

It's a bit moot right now, since there's just one tool -- but I now lean toward keeping per-tool enables rather than lumping.